mdsuite/calculators/einstein_distinct_diffusion_coefficients.py

```diff
@@ -288,8 +355,8 @@ def _post_operation_processes(self, species: Union[str, tuple] = None):
             error = np.sqrt(np.diag(covariance))[0]
 
             data = {
-                self.result_keys[0]: -1 / 6 * fit_values[0],
-                self.result_keys[1]: 1 / 6 * error,
+                self.result_keys[0]: -1 / 2 * fit_values[0],
```

Comment:
Why did this prefactor change?

Reply:
During the mapping I average over the dimension. Therefore I removed the dimension term here. I don't like this too much and we can just sum and leave the 6 here, what do you think?

Reply:
actually I like it more with the 2 here. But for these equations, a quick comment with the corresponding formulae would be nice so one can see where the numbers come from

Reply:
Yep fully agree, I will add that